openbas-front/src/utils/api-types.d.ts

```diff
@@ -2678,50 +2678,46 @@ export interface RenewTokenInput {
 }
 
 export interface Report {
-  listened?: boolean;
+  exercise?: Exercise;
```

Comment:
A report can exist without exercise ?

Reply:
I may have been thinking too much about the future, considering that a report could be associated not only with an exercise but also with challenges or other things.